LICENSE

```diff
@@ -1,13 +1,292 @@
-Copyright 2016 Intel Corporation
```

Comment:
I think this copyright notice should remain 
See around line 189 "... To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information. (Don't include the brackets!) ... Copyright [yyyy] [name of copyright owner]"

Reply:
That is in the section that provides directions on how to apply the license to "your work"; that is, all the other files you've created, not the license itself.  See for example how its handled in this major Apache-sponsored project: https://github.com/apache/spark/blob/master/LICENSE

Reply:
... on which was used as an example for this revision

Reply:
ok, makes sense